core/trino-main/src/main/java/io/trino/sql/analyzer/FeaturesConfig.java

```diff
@@ -1037,15 +1025,15 @@ public FeaturesConfig setPlanWithTableNodePartitioning(boolean planWithTableNode
         return this;
     }
 
-    public JoinPushdownMode getJoinPushdownMode()
+    public boolean isEnableJoinPushdown()
     {
-        return joinPushdownMode;
+        return enableJoinPushdown;
     }
 
-    @Config("optimizer.join-pushdown")
-    public FeaturesConfig setJoinPushdownMode(JoinPushdownMode joinPushdownMode)
+    @Config("optimizer.enable-join-pushdown")
```

Comment:
`optimizer.join-pushdown.enabled` seems to be 'the standard' for config toggles.


... maybe  we'll not only enable this by default, but actually remove it from engine side?
we don't have safety toggles for agg pushdown for instance

Reply:
If you look at this class it would be `optimizer.join-pushdown-enabled`. But I agree that we can drop it and leave just configuraiton on connector side.

Reply:
dropped